File: ts/iq/md~get~ref.ts
/** <iq type="get" xmlns="md">. Replies with one of: Success. */
export function buildMdGetRef(): BinaryNode {
  return { tag: 'iq', attrs: { to: 's.whatsapp.net', xmlns: 'md', type: 'get' }, content: [
      { tag: 'ref', attrs: {  } },
    ] }
}

File: rust/iq/md~get~ref.rs
/// <iq type="get" xmlns="md">. Replies with one of: Success.
pub fn build_md_get_ref() -> Node {
    NodeBuilder::new("iq")
        .attr("to", "s.whatsapp.net")
        .attr("xmlns", "md")
        .attr("type", "get")
        .child(NodeBuilder::new("ref").build())
        .build()
}